example/presto_standalone/README.md

```diff
@@ -0,0 +1,32 @@
+# Presto standalone example
+The current directory contains terraform related files that use the module in `../../`. The example module spins up presto in [standalone mode](../conf/nomad/presto_standalone.hcl
+) having one node work as both coordinator and worker.
+It uses vault as the shared secret provider. For more details check [main.tf](./main.tf).
```

Comment:
```suggestion
The current directory contains terraform related files that use the module in `../../`. The example module spins up Presto in [standalone mode](../conf/nomad/presto_standalone.hcl
) having one node work as both coordinator and worker.
It uses Vault as the shared secret provider. For more details check [main.tf](./main.tf).
```

Reply:
Added a new commit that fixed all caps-stuff https://github.com/fredrikhgrelland/terraform-nomad-presto/pull/74/commits/e54d93616741e07f25a19754c78332943df61aed